src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      class="search-bar form-big"
+      action="/search"
+      method="get"
+      data-qa="big"
+    >
+
+      <img
+        class="image image__big"
+        src="./images/Search.svg"
+        alt="search"
+      >
+      <input
+        class="search__input input-big"
+        data-qa="keypress"
+        type="search"
+        placeholder= "Try &quot;Los Angeles&quot;"
+        name="search"
+      >
+    </form>
+
+
+    <form
+      class="search-bar form-small"
+      action="/search"
+      method="get"
+      data-qa="small"
+    >
+      <img
+        class="image image__small"
+        src="./images/Search.svg"
+        alt="search"
+      >
+      <input
+        class="search__input input-small"
+        data-qa="small"
+        type="search"
+        placeholder= "Try &quot;Los Angeles&quot;"
+        name="search">
```

Comment:
Not fixed 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/70009205/8e17cbc5-8069-4385-8b8b-9a4f3e830878)